src/components/DeleteConfirmationModal/DeleteConfirmationModal.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { FunctionComponent } from 'react';
+import { Button, Checkbox, List, ListItem, Modal, ModalVariant } from '@patternfly/react-core';
+import { getConfig } from '../../utilities/platformServices';
+import Cookies from 'js-cookie';
+
+interface DeleteConfirmationModalProps {
+  handleModalToggle: () => void;
+  successCallback: () => void;
+  isOpen: boolean;
+  name: string;
+  uuid: string;
+}
+
+const DeleteConfirmationModal: FunctionComponent<DeleteConfirmationModalProps> = ({
+  handleModalToggle,
+  successCallback,
+  isOpen,
+  name,
+  uuid
+}) => {
+  const [confirmed, setConfirmed] = React.useState(false);
+
+  const deleteManifest = () => {
+    const jwtToken = Cookies.get('cs_jwt');
+    const { rhsmAPIBase } = getConfig();
+    fetch(`${rhsmAPIBase}/management/v1/allocations/${uuid}?force=true`, {
+      method: 'DELETE',
+      headers: { Authorization: `Bearer ${jwtToken}` },
+      mode: 'cors'
+    }).then((response) => {
+      if (response.status != 200) {
+        throw new Error(
+          `Status Code ${response.status}.  Error deleting manifest: ${response.statusText}.`
+        );
+      }
+      successCallback();
```

Comment:
Yeah, I can try that.  I just hate having to make another query when it isn't needed, but I agree it would make the code cleaner (and guarantee the table matches reality in the off chance the manifest isn't actually deleted by the api).

Reply:
This is interesting.  I'll test that out Friday.  Thanks.

Reply:
It works!  I'm going to try adding a spinner back into the modal again per your other suggestion and see if that looks good now that there isn't a second load.